llama_stack/providers/inline/inference/sentence_transformers/config.py

```diff
@@ -7,4 +7,8 @@
 from pydantic import BaseModel
 
 
-class SentenceTransformersInferenceConfig(BaseModel): ...
+class SentenceTransformersInferenceConfig(BaseModel):
+
+    @classmethod
+    def sample_run_config(cls) -> "SentenceTransformersInferenceConfig":
```

Comment:
this signature is wrong. it should be `-> dict:`

Reply:
done.